cli/command/image/remove_test.go

```diff
@@ -68,15 +77,6 @@ func TestNewRemoveCommandSuccess(t *testing.T) {
 			},
 		},
 		{
-			name: "Image Deleted with force option",
```

Comment:
Possibly. The problem with exporting them is that other packages could start to use them to report errors, which is not really the intent.